tests/integration/pilot/gw_topology_test.go

```diff
@@ -185,33 +186,8 @@ spec:
 				return err
 			}, retry.Timeout(time.Minute*2), retry.Delay(time.Second))
 
-			t.NewSubTest("nofilter").Run(func(t framework.TestContext) {
-				for _, tt := range common.ProxyProtocolFilterNotAppliedGatewayCase(&apps, fmt.Sprintf("custom-gateway.%s.svc.cluster.local", gatewayNs.Name())) {
```

Comment:
nit: I think this would leave `ProxyProtocolFilterNotAppliedGatewayCase` unused, maybe able to clean it up

Reply:
Yeah, that test is not very useful IMHO. It simply tests TCP proxying works which has nothing to do with gateway topology. Will remove.